Recrutify/Recrutify.Services/Services/ScheduleService.cs

```diff
@@ -47,5 +56,34 @@ public async Task<ScheduleDTO> GetByDatePeriodForCurrentUserAsync(DateTime? date
             var schedules = await _scheduleRepository.GetByDatePeriodAsync(userId, date.Value, daysNum);
             return _mapper.Map<ScheduleDTO>(schedules);
         }
+
+        public async Task UpdateScheduleSlotsForCurrentUserAsync(IEnumerable<DateTime> dates)
+        {
+            var currentUserId = _userProvider.GetUserId();
+            var periodStartDate = dates.Min();
+            var periodFinishDate = periodStartDate.Date.AddDays(Constants.Week.CountDays - (int)periodStartDate.DayOfWeek + 1);
```

Comment:
If you use this constant only in service, move to services constants from data access

Reply:
+